model/assigned_vm_address.rb

```diff
@@ -5,6 +5,11 @@
 class AssignedVmAddress < Sequel::Model
   one_to_one :vm, key: :dst_vm_id
   many_to_one :address, key: :address_id
+  one_to_one :active_billing_record, class: :BillingRecord, key: :resource_id, conditions: {Sequel.function(:upper, :span) => nil}
 
   include ResourceMethods
+
+  def before_destroy
```

Comment:
I think we should stop billing as soon as we get the request (maybe not in endpoint, but as a first step in deprovisioning, so that we don't charge customers unnecessarily if deprovisioning took look time for some reason.

Reply:
Maybe we can move to same place where we stop vm billing?

Reply:
I moved it